tests/moduletool/common.py

```diff
@@ -100,6 +101,64 @@ def add_file(modpath, file, content, msg, version=None, dev=False, tag=True):
         os.chdir(old_cwd)
 
 
+def add_requires(
+    modpath: str, deps: List[Tuple[str, str]], commit_msg: str, version: str, dev: bool = False, tag: bool = True
+) -> None:
+    """
+    Add the version requirements of dependencies in a module's YAML file and adds the import to the .cf file.
+    Performs a git commit and tags the commit with the specified version.
+
+    :param modpath: The path to the module.
+    :param deps: A list of tuples, each containing a dependency name and its corresponding version specification.
+    :param commit_msg: The commit message to use
+    :param version: The version to tag the commit with
+    :param dev: A flag indicating whether this is a development version. Default is False.
+    :param tag: A flag indicating whether to tag the commit. Default is True.
+    """
+    mainfile = "module.yml"
+    file_path = os.path.join(modpath, mainfile)
+    yaml = ruamel.yaml.YAML()
+
+    with open(file_path, "r") as file:
+        data = yaml.load(file)
+
+    # Ensure 'requires' field exists and is a list
+    if "requires" not in data:
+        data["requires"] = []
+
+    # Prepare a dictionary to hold the latest version requirement for each module
+    requires_dict = {item.split()[0]: item for item in data["requires"]}
```

Comment:
This is a bit more robust.
```suggestion
    requires_dict = {item.strip().split()[0]: item for item in data["requires"]}
```